### DISCUSSION TITLE
Welcome to Support Discussions!

### DISCUSSION THREAD

**Discussion by kartikver15gr8** [BuildersChain/Support#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by im-vishalanand** [comment]
Hii. My name is Vishal, and I would love to be part of it and to contribute along with new learnings.

**Reply by yashdev9274** [reply]
Glad to here it 
Make sure to invite yourself to github org and get started by contributing on projects and  get involved in community conversations

**Comment by devjayswal** [comment]
Dear BuildersChain Community Head Kartikey Verma Sir and Yash Sir,

I hope this message finds you in good health and high spirits. I am writing to express my interest in becoming a Pull Request Reviewer for BuildersChain on GitHub.

As an avid contributor to BuildersChain and someone with a keen interest in the project, I would like to take on a more active role in ensuring the quality and consistency of code contributions made by other members of the community. I believe that by becoming a Pull Request Reviewer, I can help ensure that the project's codebase remains stable, secure, and efficient.

I understand that becoming a Pull Request Reviewer requires a deep understanding of the BuildersChain project and the underlying technologies it employs. To that end, I have invested significant time and effort into familiarizing myself with the codebase, and I believe that I am well-equipped to help review and merge incoming pull requests.

If you agree that my skills and expertise would be valuable to the BuildersChain community as a Pull Request Reviewer, I would be grateful for the opportunity to serve in this capacity. Please let me know if there are any further steps that I need to take or if there is any additional information that you require.

Thank you for your time and consideration.

Sincerely,
Dev jayswal
To
@kartikver15gr8 
@yashdev9274 
```
```

```
```
